lib/constants.dart

```diff
@@ -1,5 +1,5 @@
-class AppConstsnts {
+class AppConstants {
   static const String endPoint = "https://demo.appwrite.io/v1";
```

Comment:
@brandonroberts, what do you think about changing this URL. Our default values in this code will actually connect to our working demo instance/project.